.github/workflows/main.yml

```diff
@@ -54,9 +54,6 @@ jobs:
       - name: install dependencies
         run: pnpm install
 
-      - name: build packages
```

Comment:
why did you remove a build step from CI?

Reply:
I assumed you only had it there to enable `pnpm test` to run without problems - because of the path aliases I added to `vitest.config.ts`, the build step is not required for tests anymore.

Reply:
no, it builds in CI to ensure that building packages doesn't fail

Reply:
I'll revert, then